worker/categorize.py

```diff
@@ -387,8 +376,7 @@ def _get_best_keywords(self, communities, original_keywords):
             best_original_keywords = keyword_counts.most_common(2)
             best_keywords.update([keyword[0] for keyword in best_original_keywords])
 
-        return [keyword.title() for keyword in best_keywords] if best_keywords else [""]
-
+        return [keyword.title() for keyword in best_keywords]
```

Comment:
Sounds good. Can you point me to the .categorized check in the code when you get a chance?